addons/survey/static/src/js/survey_session_chart.js

```diff
@@ -229,8 +229,13 @@ publicWidget.registry.SurveySessionChart = publicWidget.Widget.extend({
                         let resultLines = [];
                         let currentLine = [];
                         for (let i = 0; i < words.length; i++) {
-                            // If the word we are adding exceed already the number of characters for the line, we add it anyway before passing to a new line
-                            currentLine.push(words[i]);
+                            // Chop down words that do not fit on a single line, add each part on its own line.
+                            let word = words[i];
+                            while (word.length > charPerLine) {
+                                resultLines.push(word.slice(0, charPerLine - 1) + '-');
+                                word = word.slice(charPerLine - 1);
+                            }
+                            currentLine.push(word);
```

Comment:
Would be nice to adapt the docstring to mention that we do this, maybe with a small example.

Reply:
Added:
> When the screen space is too small for long words, those long words are splitt over multiple rows.
At 6 chars per row, the above example becomes `["this", "is an", "examp-", "le of", "a label"]`
Which is displayed as `"this<br/>is an<br/>examp-<br/>le of<br/>a label"`